_layouts/default.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="{{ site.lang | default: "en-US" }}">
```

Comment:
Can we change default local from `en-US` to `en-GB`?

```html
<html lang="{{ site.lang | default: "en-GB" }}">
```

Reply:
Cool stuff @x4d3, everything is good and I added a suggestion

Reply:
Thanks, I would do the the change  in `_config.yml` though, trying to keep the template as close as possible as the original one.

Reply:
Also don't hesitate to use the amazing feature of github: insert a suggestion :)

![image](https://user-images.githubusercontent.com/4570217/104844827-42a9e280-58ca-11eb-9c53-056b0851250a.png)

Reply:
@pataruco I've changed _config.yml to add `lang=en` so there is no 🇺🇸  vs 🇬🇧  debate :)

Reply:
Amazing!